diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back-end is where things happens behind the scenes ... it is were the processing occurs ..
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+the controller is the "C" in the model
```

Comment:
In MVC M stands for model and C stands for controller. For this question we are asking which layer does the controller use to fetch data. The correct answer is model.